appveyor.yml

```diff
@@ -2,49 +2,40 @@
 # http://www.appveyor.com/docs/installed-software#python
 
 # This configuration based on:
-# https://github.com/audreyr/cookiecutter/commit/3c4685f536afda3be93da3fe3039cec0ab0d60a3
+# https://github.com/cookiecutter/cookiecutter/blob/5e65edf4c340993f462ddeaf44f99eb6f9da66f9/appveyor.yml
 
 environment:
   matrix:
     - PYTHON: "C:\\Python27-x64"
-      TOX_ENV: "py27"
+      TOX_ENV: "test-py27,codecov"
+
+    - PYTHON: "C:\\Python35-x64"
+      TOX_ENV: "test-py35,codecov"
 
     - PYTHON: "C:\\Python36-x64"
-      TOX_ENV: "py36"
+      TOX_ENV: "test-py36,codecov"
+
+    - PYTHON: "C:\\Python37-x64"
+      TOX_ENV: "test-py37,codecov"
 
 
 init:
+  - set OS=WINDOWS
   - set PATH=%PYTHON%;%PYTHON%\Scripts;%PATH%
   - "git config --system http.sslcainfo \"C:\\Program Files\\Git\\mingw64\\ssl\\certs\\ca-bundle.crt\""
   - "%PYTHON%/python -V"
   - "%PYTHON%/python -c \"import struct;print(8 * struct.calcsize(\'P\'))\""
```

Comment:
I wonder what the red highlighting in the diff here is about…

Reply:
I think it thinks something's off with the escaping. I've seen it around from time to time, doesn't seem to be correlated with actual breakage so I haven't given it much more thought lol